src/settings-ui/Settings.UI/SettingsXAML/Views/DashboardPage.xaml

```diff
@@ -0,0 +1,385 @@
+                                                                <ItemsPanelTemplate>
+                                                                    <StackPanel Orientation="Horizontal" Spacing="12" />
+                                                                </ItemsPanelTemplate>
+                                                            </ItemsControl.ItemsPanel>
+                                                            <ItemsControl.ItemTemplate>
+                                                                <DataTemplate>
+                                                                    <custom:KeyVisual
+                                                                        VerticalAlignment="Center"
+                                                                        AutomationProperties.AccessibilityView="Raw"
+                                                                        Content="{Binding}"
+                                                                        IsTabStop="False"
+                                                                        VisualType="TextOnly" />
+                                                                </DataTemplate>
+                                                            </ItemsControl.ItemTemplate>
+                                                        </ItemsControl>
+                                                    </Border>
+                                                    <TextBlock
+                                                        Grid.Column="1"
+                                                        VerticalAlignment="Center"
+                                                        Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                                                        Text="{x:Bind Label, Mode=OneWay}"
+                                                        TextWrapping="WrapWholeWords" />
+                                                </Grid>
+                                                <controls:SettingsCard
+                                                    Padding="8"
+                                                    MinHeight="0"
+                                                    Click="{x:Bind ButtonClickHandler, Mode=OneWay}"
+                                                    Header="{x:Bind ButtonTitle, Mode=OneWay}"
+                                                    IsClickEnabled="True"
+                                                    IsActionIconVisible="False"
+                                                    Visibility="{x:Bind IsButtonVisible, Converter={StaticResource BoolToVisibilityConverter}, Mode=OneWay}">
+                                                    <controls:SettingsCard.Resources>
+                                                        <x:Double x:Key="SettingsCardHeaderIconMaxSize">14</x:Double>
```

Comment:
Why 14? That's not a multiple of 4.

Reply:
This code has been deleted meanwhile. Thanks!